### PR TITLE
Remove maker useless option

### PR DESCRIPTION
**Pull request by nawel-les-tilleuls**
Removed `maker` useless option

### CODE REVIEW

File: src/Api/FilterInterface.php [hunk 1]
```diff
@@ -45,7 +45,7 @@ interface FilterInterface
      *     ]
      * The description can contain additional data specific to a filter.
      *
-     * @see \ApiPlatform\Core\Swagger\Serializer\DocumentationNormalizer::getFiltersParameters
+     * @see \Serializer\Filter\PropertyFilter::getDescription
```
Comment (vincentchalamon) [inline src/Api/FilterInterface.php:48]:
```suggestion
     * @see \ApiPlatform\Serializer\Filter\PropertyFilter::getDescription
```

**Review by vincentchalamon — CHANGES_REQUESTED** [review]
Minor change, then LGTM :+1:
I'm just not 100% sure about the maker section deletion: was it deprecated in 2.7? Should we keep it in 3.0 to add (even empty) new commands on it? WDYT @soyuka?

**Review by vincentchalamon — APPROVED** [review]
LGTM (waiting for confirmation about the maker section)

**Comment by dannyvw** [comment]
See https://github.com/api-platform/core/pull/4827 for more changes. Also core namespace is removed.

**Comment by vincentchalamon** [comment]
Closing as a Maker command is about to be added here: https://github.com/api-platform/core/pull/4423

**Pull request closed without merging**